src/plugins/intel_npu/src/backend/src/zero_infer_request.cpp

```diff
@@ -529,14 +532,14 @@ void ZeroInferRequest::infer_async() {
     _logger.debug("InferRequest::infer_async started");
     OV_ITT_TASK_CHAIN(ZERO_INFER, itt::domains::LevelZeroBackend, "infer_async", "start");
 
-    _executor->mutexLock();
+    _graph->mutexLock();
```

Comment:
This is not exception safe, use lock guard.

Reply:
understand, thx for pointing this out. changed.